commit b7e0f286a2ecab3b687ec9b3f95f5a88b9f85310
Author: Orlando Arias <oarias@knights.ucf.edu>
Date:   Fri Jan 19 11:45:01 2018 -0500

    Fix syntax error in exit.c
    
    This patch fixes a syntax error in exit.c that was introduced during the
    ANSI-fication of newlib. The patch fixes a compile-time issue that arises when
    newlib is configured with the --enable-lite-exit feature.

diff --git a/newlib/libc/stdlib/exit.c b/newlib/libc/stdlib/exit.c
index 95108ade6..3e618914e 100644
--- a/newlib/libc/stdlib/exit.c
+++ b/newlib/libc/stdlib/exit.c
@@ -54,7 +54,7 @@ exit (int code)
 {
 #ifdef _LITE_EXIT
   /* Refer to comments in __atexit.c for more details of lite exit.  */
-  void __call_exitprocs (int, void *)) __attribute__((weak);
+  void __call_exitprocs (int, void *) __attribute__((weak));
   if (__call_exitprocs)
 #endif
     __call_exitprocs (code, NULL);
